prog/vm/host_nexus.rb

```diff
@@ -104,6 +105,43 @@ def wait_setup_spdk
   end
 
   def wait
+    when_reboot_set? do
+      hop :reboot
+    end
+
     nap 30
   end
+
+  def reboot
+    vm_host.vms.each { |vm|
+      vm.update(display_state: "rebooting host")
+    }
+
+    begin
+      sshable.cmd("sudo reboot")
+    rescue
+    end
```

Comment:
Hmmm, sure seems like stuff can go wrong here, and you wouldn' tget the reboot you intend (e.g.  an IOError preventing sending the rebot). I know from experience there's a bit of sensitivity here because systemd is so fast at killing SSH abruptly.

Reply:
In my previous experience with restarts, it was basically necessary to record something like a boot identifier (this can be time, and I think systemd has an entropic one as well), to prove whether the reboot happened. I wish I had a good implication for you, but I have found no abbreviation for going at it in a rather brutal and overwrought seeming way, given the apparent simplicity given the task at hand.

(e.g:
```
journalctl --list-boots 
IDX BOOT ID                          FIRST ENTRY                 LAST ENTRY                 
-25 3f8abb5def0e4d7889d72a48a0c0772e Sun 2023-06-18 09:31:19 PDT Sun 2023-06-18 09:33:50 PDT
``)

Reply:
```sh
$ cat  /proc/sys/kernel/random/boot_id
af4c5dab-3050-4868-aa12-ecb0c6151677
```

Reply:
Done, and also now using `sudo systemctl reboot` which is async and doesn't close the ssh connection immediately. It reboots just after we receive an ssh reply.